Tests/FunctionalTests/AEPExperiencePlatformFunctionalTests.swift

```diff
@@ -80,19 +80,19 @@ class AEPExperiencePlatformFunctionalTests: FunctionalTestBase {
     // MARK: test request event format
 
     func testSendEvent_withXDMData_sendsCorrectRequestEvent() {
-        setExpectationEvent(type: FunctionalTestConst.EventType.EXPERIENCE_PLATFORM, source: FunctionalTestConst.EventSource.REQUEST_CONTENT)
+        setExpectationEvent(type: FunctionalTestConst.EventType.EXPERIENCE, source: FunctionalTestConst.EventSource.REQUEST_CONTENT)
 
-        let experienceEvent = ExperiencePlatformEvent(xdm: ["testString": "xdm",
-                                                            "testInt": 10,
-                                                            "testBool": false,
-                                                            "testDouble": 12.89,
-                                                            "testArray": ["arrayElem1", 2, true],
-                                                            "testDictionary": ["key": "val"]])
-        ExperiencePlatform.sendEvent(experiencePlatformEvent: experienceEvent)
+        let experienceEvent = ExperienceEvent(xdm: ["testString": "xdm",
+                                                    "testInt": 10,
+                                                    "testBool": false,
+                                                    "testDouble": 12.89,
+                                                    "testArray": ["arrayElem1", 2, true],
+                                                    "testDictionary": ["key": "val"]])
+        Edge.sendEvent(ExperienceEvent: experienceEvent)
```

Comment:
`Edge.sendEvent(experienceEvent: experienceEvent)` - same for all usages bellow

Reply:
whoops, bad find & replace.